charts/rasa-x/templates/rasa-secret.yaml

```diff
@@ -11,4 +11,6 @@ data:
   rasaXToken: {{ .Values.rasax.token | b64enc | quote }}
   passwordSalt: {{ .Values.rasax.passwordSalt | b64enc | quote }}
   jwtSecret: {{ .Values.rasax.jwtSecret | b64enc | quote }}
+  initialUserName: {{ .Values.rasax.initialUser.username | b64enc | quote }}
```

Comment:
but why would we put it in the secret? Other charts also don't do this (e.g. https://github.com/helm/charts/blob/master/stable/postgresql/templates/secrets.yaml)

Reply:
the official guide says that it's fine to put username and password there (https://kubernetes.io/docs/concepts/configuration/secret/), so I am not sure what is the best practice. I will investigate and update you :)

Reply:
You were right — it's better to remove it from there. I did it and pushed — so please check it again.